cookie-banner-rules-list.json

```diff
@@ -7461,6 +7461,25 @@
         ]
       },
       "id": "637c8bb9-b2f6-493e-86c2-f1954aa6a96a"
+    },
+    {
+      "click": {
+        "optIn": "button[name=accept_cookie]",
+        "optOut": "button[name=decline_cookie]",
+        "presence": ".js-cookie-notification__form"
+      },
+      "domain": "coolblue.nl",
+      "cookies": {
+        "optOut": [
+          {
+            "name": "cookie-preferences",
+            "path": "/",
+            "value": "eyJmdW5jdGlvbmFsIjp0cnVlLCJhbmFseXRpY2FsIjp0cnVlLCJtYXJrZXRpbmciOmZhbHNlfQ%3D%3D",
+            "sameSite": 0
```

Comment:
nit: These should be the default values for `sameSite` and `path` so I think they can be omitted.

Reply:
Ah, just copied that from other rules. Fixed in b5fcc3b.